src/tools/pontoon/ui.md

```diff
@@ -191,6 +191,23 @@ The origin of the machinery entry is listed in gray above the source string. Cli
 
 Be careful when using the machinery tab as suggestions may not match the source string for the project being translated. Even if the source strings match, the context in which strings are used may not be the same. This could lead to incorrect or unnatural translations. Always keep the meaning and purpose of the string being translated in mind when using the machinery tab.
 
+#### Large language model (LLM) integration
+
+For all locales that use Google Translate as a translation source, a dropdown labeled `AI` becomes available. This feature refines the Google Translate output using an LLM. Collapsing this dropdown will reveal three options:
+* `REPHRASE`: Generate an alternative to this translation.
+* `MAKE FORMAL`: Generate a more formal version of this translation.
+* `MAKE INFORMAL`: Generate a more informal version of this translation.
+
+After selecting an option, the revised translation will replace the original suggestion. Once a new translation is generated, another option `SHOW ORIGINAL` will be seen in the dropdown menu. You can revert to the original translation by selecting the `SHOW ORIGINAL` option from the dropdown menu.
```

Comment:
```suggestion
After selecting an option, the revised translation will replace the original suggestion. Once a new translation is generated, another option `SHOW ORIGINAL` will be available in the dropdown menu. Selecting it will revert to the original suggestion.
```